lib/simplecov_small_badge/formatter.rb

```diff
@@ -33,8 +33,8 @@ def percent_text(percent)
     end
 
     def state(covered_percent)
-      if line_coverage_minimum&.positive?
-        if covered_percent >= line_coverage_minimum
+      if SimpleCov.minimum_coverage[:line]&.positive?
+        if covered_percent >= SimpleCov.minimum_coverage[:line]
```

Comment:
```suggestion
      if coverage_minimum&.positive?
        if covered_percent >= coverage_minimum
```

With:
```ruby
    def coverage_minimum
      @coverage_minimum ||= begin
        minimums = SimpleCov.minimum_coverage
        minimums.is_a?(Hash) ? minimums[SimpleCov.primary_coverage] : minimums
      end
    end
```

This ensures that we use the minimum corresponding to the primary coverage (which is not necessary `:line`)